tools/api_proto_plugin/plugin.py

```diff
@@ -88,9 +86,9 @@ def plugin(output_descriptors):
             stats_file.name = file_proto.name + '.profile'
             ps.print_stats()
             stats_file.content = stats_stream.getvalue()
-        # Also include the original FileDescriptorProto as text proto, this is
-        # useful when debugging.
-        descriptor_file = response.file.add()
-        descriptor_file.name = file_proto.name + ".descriptor.proto"
-        descriptor_file.content = str(file_proto)
+            # Also include the original FileDescriptorProto as text proto, this is
+            # useful when debugging.
+            descriptor_file = response.file.add()
```

Comment:
This has been moved underneath the `cprofile_enabled`?

Reply:
yep - for testing purposes

not sure what is best here - i think in the main we dont want the io of creating this when its not used (iiuc it mostly isnt)

perhaps we can add another os.environ flag - `DUMP_PROTO_DESCRIPTORS` or similar